### PR TITLE
Public cloud container deployment update

### PR DESCRIPTION
**Pull request by jcayouette**
# Some hints
Updates public cloud guide with new container / image deployment and updates mgr-storage-server and mgr-storage-proxy commands.

- [x] Master

### CODE REVIEW

File: modules/specialized-guides/pages/public-cloud-guide/payg/aws/payg-aws-server-setup.adoc [hunk 1]
```diff
@@ -10,57 +10,39 @@
...
-zypper update
+transactional-update
 ----
 
-. Configure storage with the **suma-storage** tool. 
-** For help see: `suma-storage --help`
-This tool simplifies creating the storage disk device and database device.
+. Reboot the system.
+
+. Configure storage with the `mgr-storage-server` tool or in the case of the Proxy use `mgr-storage-proxy`.
```
Comment (keichwa) [inline modules/specialized-guides/pages/public-cloud-guide/payg/aws/payg-aws-server-setup.adoc:18]:
```suggestion
. Configure storage with the `mgr-storage-server` tool or in the case of the proxy use [command]``mgr-storage-proxy``. 
```

File: modules/specialized-guides/pages/public-cloud-guide/payg/aws/payg-aws-server-setup.adoc [hunk 2]
```diff
@@ -10,57 +10,39 @@
...
+transactional-update
 ----
 
-. Configure storage with the **suma-storage** tool. 
-** For help see: `suma-storage --help`
-This tool simplifies creating the storage disk device and database device.
+. Reboot the system.
+
+. Configure storage with the `mgr-storage-server` tool or in the case of the Proxy use `mgr-storage-proxy`. 
+** For help see: `mgr-storage-server --help` or `mgr-storage-proxy --help`.
```
Comment (keichwa) [inline modules/specialized-guides/pages/public-cloud-guide/payg/aws/payg-aws-server-setup.adoc:19]:
```suggestion
** For more information, see [command]``mgr-storage-server --help`` or [command]``mgr-storage-proxy --help``.
```

File: modules/specialized-guides/pages/public-cloud-guide/payg/aws/payg-aws-server-setup.adoc [hunk 3]
```diff
@@ -10,57 +10,39 @@
...
 
 [NOTE]
 ====
-This command will create the following storage locations:
-
-----
-/pgsql_storage
-/pgsql_storage/pgsql
-----
+This command will create the following persistent storage volumes at `/var/lib/containers/storage/volumes`:
```
Comment (keichwa) [inline modules/specialized-guides/pages/public-cloud-guide/payg/aws/payg-aws-server-setup.adoc:39]:
```suggestion
This command will create the following persistent storage volumes at [path]``/var/lib/containers/storage/volumes``:
```

File: modules/specialized-guides/pages/public-cloud-guide/payg/aws/payg-aws-server-setup.adoc [hunk 4]
```diff
@@ -10,57 +10,39 @@
...
-----
-/manager_storage
-/manager_storage/spacewalk
-----
-
-----
-/var/spacewalk
-/var/cache/rhn
-/var/lib/spacewalk
+See xref:installation-and-upgrade:container-management/persistent-container-volumes.adoc[]
```
Comment (keichwa) [inline modules/specialized-guides/pages/public-cloud-guide/payg/aws/payg-aws-server-setup.adoc:41]:
```suggestion
For more information, see xref:installation-and-upgrade:container-management/persistent-container-volumes.adoc[].
```

File: modules/specialized-guides/pages/public-cloud-guide/payg/azure/payg-azure-server-setup.adoc [hunk 5]
```diff
@@ -51,53 +51,44 @@ Additionally, be mindful of the geographical restrictions associated with your p
...
+sudo -i
+transactional-update
+----
+
+. Reboot the system.
 
-. Use `lsblk` to see the names of your disk devices.
-You will see a list of partitions that were created when we configured the {azure} image.
+. Configure storage with the `mgr-storage-server` tool or in the case of the Proxy use `mgr-storage-proxy`. 
+For help see: `mgr-storage-server --help` or `mgr-storage-proxy --help`.
```
Comment (keichwa) [inline modules/specialized-guides/pages/public-cloud-guide/payg/azure/payg-azure-server-setup.adoc:67]:
```suggestion
For more information, see [command]``mgr-storage-server --help`` or [command]``mgr-storage-proxy --help``.
```

File: modules/specialized-guides/pages/public-cloud-guide/payg/azure/payg-azure-server-setup.adoc [hunk 6]
```diff
@@ -51,53 +51,44 @@ Additionally, be mindful of the geographical restrictions associated with your p
...
+----
+sudo -i
+transactional-update
+----
+
+. Reboot the system.
 
-. Use `lsblk` to see the names of your disk devices.
-You will see a list of partitions that were created when we configured the {azure} image.
+. Configure storage with the `mgr-storage-server` tool or in the case of the Proxy use `mgr-storage-proxy`.
```
Comment (keichwa) [inline modules/specialized-guides/pages/public-cloud-guide/payg/azure/payg-azure-server-setup.adoc:66]:
```suggestion
. Configure storage with the [command]``mgr-storage-server`` tool or in the case of the proxy use [command]``mgr-storage-proxy``. 
```

File: modules/specialized-guides/pages/public-cloud-guide/payg/azure/payg-azure-server-setup.adoc [hunk 7]
```diff
@@ -51,53 +51,44 @@ Additionally, be mindful of the geographical restrictions associated with your p
...
 
-suma-storage /dev/nvme1n1 /dev/nvme2n1
+mgr-storage-server /dev/nvme1n1 /dev/nvme2n1
 ----
 +
 
 [NOTE]
 ====
-This command will create the following storage locations:
+This command will create the following persistent storage volumes at `/var/lib/containers/storage/volumes`:
```
Comment (keichwa) [inline modules/specialized-guides/pages/public-cloud-guide/payg/azure/payg-azure-server-setup.adoc:85]:
```suggestion
This command will create the following persistent storage volumes at [path]``/var/lib/containers/storage/volumes``:
```

File: modules/specialized-guides/pages/public-cloud-guide/payg/azure/payg-azure-server-setup.adoc [hunk 8]
```diff
@@ -51,53 +51,44 @@ Additionally, be mindful of the geographical restrictions associated with your p
...
-----
-/manager_storage
-/manager_storage/spacewalk
-----
-
-----
-/var/spacewalk
-/var/cache/rhn
-/var/lib/spacewalk
+See xref:installation-and-upgrade:container-management/persistent-container-volumes.adoc[]
```
Comment (keichwa) [inline modules/specialized-guides/pages/public-cloud-guide/payg/azure/payg-azure-server-setup.adoc:87]:
```suggestion
For more information, see xref:installation-and-upgrade:container-management/persistent-container-volumes.adoc[].
```

**Review by keichwa — APPROVED** [review]

**Review by admd — APPROVED** [review]
Thank you Joseph. Content wise it looks good to be but we need to get it verified from @maximenoel8 or @mcalmer.

**Pull request merged**